src/btree/bt_npos.c

```diff
@@ -0,0 +1,373 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include "wt_internal.h"
+
+/*
+ * !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+ *
+ * Below is a set of functions for computing the normalized position of a page and restoring a page
+ * from its normalized position. These functions are used by the eviction server for the sake of
+ * not holding the hazard pointer for longer than necessary. Another user is "partition cursor".
```

Comment:
Please remove the mention of partition cursor as it is not implemented

Reply:
Done.